site/content/blog/2022-12-24-12-days-of-zapmas/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "The Twelve Days of ZAPmas"
+summary: >
+  A reply to an excellent blog series from [Secure Ideas](https://www.secureideas.com/): Twelve Days of ZAPmas - ZAP impressions from a Burp user.
```

Comment:
Should we stick a 2022 in there to distinguish from any that might occur int he future?

Reply:
We could add that in the future if new ZAPmas blog posts are made which coulod be confusing?

Reply:
Sure